tests/test_Imageclassification.py

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
The tests should be in https://github.com/mlflow/mlflow/blob/master/tests/transformers/test_transformers_model_export.py 
And will need to verify a number of elements, as with other existing tests for pipeline types in transformers:

1. save_model and load_model functionality
2. log_model (with registration) and load_model
3. signature inference with validation of both input_example, signature validation with pyfunc serving and scoring
4. pyfunc input types (pandas DF formatting types (records, split, inputs)) 
5. input type encodings for multiple image types to ensure that the provided encoding and decoding functions correctly
6. assertions of inference structure conformance between native pipeline loading and pyfunc inference. 
7. assurance that uri-based inference works as intended
8. validation of defined exceptions (invalid uri throws, for instance)

Reply:
Added testcases

Reply:
Can this file please be removed? The fixture can be moved to the main transformers test file. We don't need this file here.

Reply:
done